src/main/webapp/app/panel/panel.component.ts

```diff
@@ -127,9 +154,10 @@ export class PanelComponent implements OnInit {
             case 'update':
                 if (path.length === 1) {
                     if (obj[path[0]].hasOwnProperty('genomic')) {
-                        this.updateNode(obj[path[0]], 'genomic');
+                        obj[path[0]]['genomic'] = _.clone(this.genomicInput);
```

Comment:
How do you update node data after removing updateNode() method?

Reply:
resolved